models/property.model.js

```diff
@@ -3,8 +3,8 @@ const connection = require('../connections/mysql_db');
 const getAllProperty =  () => {
     let query = {
         sql: `SELECT p.*, i.imagePath FROM property as p
-              NATURAL JOIN property_photos as i
-              WHERE i.photoId = 1`
+              NATURAL JOIN property_photos as i`,
+        values: []
```

Comment:
why this was right we only need one photo

Reply:
Doesn't it for fetching all properties? Then why condition on photoId here?

Reply:
we have 4 photos for every property , when we do natural join we get every prperty 4 times with different photos , this will ensure that we got only 1 property

Reply:
yes, got it, so the primary key is combination of propertyId, and photoId in property_photos right? I misinterpreted this. I thought just photoId is primary key. Okay, I'll change it, or better, don''t merge this file.

Reply:
I don't know how to not merge a specific file.. can you tell me??

Reply:
No, I also don't know that. I'll push changed file soon.